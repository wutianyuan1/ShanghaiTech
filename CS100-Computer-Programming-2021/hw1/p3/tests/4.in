0x3f700000