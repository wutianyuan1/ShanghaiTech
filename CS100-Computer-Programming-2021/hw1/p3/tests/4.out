The number converts to 1.437500
